storagenode/piecestore/endpoint.go

```diff
@@ -218,8 +218,8 @@ func (endpoint *Endpoint) Upload(stream pb.Piecestore_UploadServer) (err error)
 		return ErrProtocol.New("expected put or put repair action got %v", limit.Action) // TODO: report grpc status unauthorized or bad request
 	}
 
-	if err := endpoint.VerifyOrderLimit(ctx, limit); err != nil {
-		return err // TODO: report grpc status unauthorized or bad request
```

Comment:
did you intend to remove this TODO?

Reply:
Yes, because if you check the changes in such method, you'll see that now it returns only gRPC errors (errors with status codes)